frontend/src/static/js/v_summary.js

```diff
@@ -385,23 +397,23 @@ window.vSummary = {
           insertions: 0,
           deletions: 0,
           date: getDateStr(startOfWeekMs),
+          endDate: getDateStr(startOfWeekMs + durationMs - DAY_IN_MS),
```

Comment:
Is it worthwhile to use endDate for all week contributes just for one special case?

Reply:
Since for ramps, we are now displaying the period of the week instead of only the start date, I guess it would be easier to have an endDate property?